src/components/EntityInfo.vue

```diff
@@ -44,14 +44,23 @@
               <v-btn
                 small text color="primary"
                 id="company-information-button"
-                :disabled="hasBlockerTask || !isInGoodStanding"
+                :disabled="hasBlocker || !isInGoodStanding"
                 @click="viewChangeCompanyInfo()"
                 @mouseenter="showHoverStyle = true"
                 @mouseleave="showHoverStyle = false"
               >
                 <v-icon medium>mdi-file-document-edit-outline</v-icon>
                 <span>View and Change Company Information</span>
               </v-btn>
+
+              <v-tooltip top content-class="pending-tooltip" v-if="!isInGoodStanding">
+                <template v-slot:activator="{ on }">
+                  <span class="pending-alert" v-on="on">
+                    <v-icon color="orange darken-2">mdi-alert</v-icon>
+                  </span>
+                </template>
+                You cannot view or change company information while the company is not in good standing.
+              </v-tooltip>
```

Comment:
I took the initiative to add this because I was initially confused why this button was disabled for an otherwise normal-looking BEN. Example:

![not in good standing tooltip](https://user-images.githubusercontent.com/10002889/134426521-230ca7e9-2a2f-4590-b2b2-0cafd26a1a7f.PNG)